lib/src/components/NxFilterInput/NxFilterInput.tsx

```diff
@@ -23,16 +26,40 @@ const NxFilterInput = forwardRef<HTMLDivElement, Props>(
           className = classnames('nx-filter-input', classNameProp, {
             'nx-filter-input--empty': isEmpty
           }),
-
+          btnClassName = classnames('nx-btn--clear', { 'hidden': isEmpty }),
+          fieldRef = useRef<HTMLDivElement>(null),
```

Comment:
Not needed anymore, which means the `mergedRef` isn't either

Reply:
Updated here 80b272ee24ecab0f0a8279530895d1bbb9098bcb. Thanks.